nixos/modules/services/misc/nzbget.nix

```diff
@@ -4,6 +4,7 @@ with lib;
 
 let
   cfg = config.services.nzbget;
+  dataDir = builtins.dirOf cfg.configFile;
```

Comment:
Now the `dataDir` option is no longer being used, but remains in the module. If the option is really to be removed you should use `mkRemovedOptionModule` and possibly mention in release notes.